build-system/app-utils.js

```diff
@@ -82,6 +86,10 @@ const replaceUrls = (mode, file, hostName, inabox, storyV1) => {
       /https:\/\/cdn\.ampproject\.org\/video-iframe-integration-v0\.js/g,
       hostName + '/dist/video-iframe-integration-v0.js'
     );
+    file = file.replace(
+      /https:\/\/cdn\.ampproject\.org\/amp-story-embed-v0\.js/g,
+      hostName + '/dist/amp-story-embed-v0.js'
```

Comment:
Is this file a part of the `amp-story` extension? If so, the path will need to follow the pattern of `dist/v0/<name><version>` in order to support lazy-building during `gulp lazy-build`.

OTOH, if it's a standalone file, is it an extension or a top-level JS file? If it's the former, add it to this list:

https://github.com/ampproject/amphtml/blob/a5640333cfc31ebc53bce54f5dc68ec5962b2192/bundles.config.js#L194-L1124

And if it's the latter, add it to this list:

https://github.com/ampproject/amphtml/blob/a5640333cfc31ebc53bce54f5dc68ec5962b2192/bundles.config.js#L35-L192

Also, add it to this list so it gets compiled during `build` and `dist`:

https://github.com/ampproject/amphtml/blob/a5640333cfc31ebc53bce54f5dc68ec5962b2192/build-system/tasks/helpers.js#L196-L219

Reply:
It's intended to be a top-level JS file that is not shipped as part of v0.js or any extension.  It will be embedded in non-AMP pages.

Reply:
Gotcha. In that case, adding it to `jsBundles` and `compileAllJs()` will do the trick.

Reply:
Thanks @rsimha!  Updated